reference/2_constructing_data/2_1_setters_deleters/README.md

```diff
@@ -0,0 +1,90 @@
+# Using setters & deleters
+
+Setters are the atomic data manipulation operations in Cranq. There are built-in nodes for dictionaries/complex types, and one for arrays.

```

Comment:
We should settle on the official term for "complex types"

Reply:
Re-phrasing to "record" everywhere